tests/test_config_linter.py

```diff
@@ -0,0 +1,152 @@
+from textwrap import dedent
+
+import pytest
+
+from ert._c_wrappers.enkf.ert_config_lint import lint_file
+from ert._c_wrappers.enkf.ert_config_lint_header import LintLocation, LintType
+
+# List of errors (from ert_config.py)
```

Comment:
What is this comment for?

Reply:
Oops, this was for development only, an attempt at enumerating all different errors that can originate, but on second look it is probably best to just go through the code and do it one by one anyway, will remove